### ISSUE TITLE
lessmsi.exe crashes on a substed drive

### ISSUE THREAD

**Issue by salsifis** [activescott/lessmsi#70]
Suppose I do the following:

```
subst K: "%HOMEPATH%\Applications"
K:\LessMSI\lessmsi.exe
```

Then it crashes. If I run, however, `%HOMEPATH%\Applications\lessmsi.exe`, then it will run smoothly.

**Comment by activescott** [comment]
No idea why that might happen. A stack trace might help point in the right direction. Would be happy to accept a pull request for this.

**Comment by salsifis** [comment]
I have the following problem on a substed drive with `lessmsi.exe x <file.msi>`:

```
Error: System.IndexOutOfRangeException:
   at LessIO.Strategies.Win32.Win32FileSystemStrategy.CreateDirectory(Path path)
   at LessMsi.Msi.Wixtracts.ExtractFiles(Path msi, String outputDir, MsiFile[] filesToExtract, AsyncCallback progressCallback)
   at LessMsi.Cli.Program.DoExtraction(String msiFileName, String outDirName, List`1 filesToExtract)
   at LessMsi.Cli.ExtractCommand.Run(List`1 allArgs)
   at LessMsi.Cli.Program.Main(String[] args)

```

This happens if I am on `D:\` which is a subst to `C:\disk_d`.

**Comment by activescott** [comment]
...still waiting on a nuget update to be able to push this fix to a full release of lessmsi...

**Comment by activescott** [comment]
Fixed in 1.5. Get it via chocolatey https://chocolatey.org/packages/lessmsi/1.5 or at https://github.com/activescott/lessmsi/releases/tag/v1.5
